### PR TITLE
Website: Responsify the rest of pegasus pages

### PR DESCRIPTION
**Pull request by tanyaparker**

### CODE REVIEW

**Comment by tanyaparker** [comment]
cc @breville

File: pegasus/sites.v3/code.org/public/3rdparty-partners.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/3rdparty-partners.md:1]: Does it make sense for the text on this page to take up the whole width on small screens?  With all the white space on the right hand side, seems like you lose valuable small screen real estate.

**Comment by Erin007** [comment]
Should we also responsify the "requested page was not found" page?

File: pegasus/sites.v3/code.org/public/certificates.haml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/certificates.haml:1]: On the iphone simulation, the text and name entry box hang off the right side and the bottom button is right up against the footer: 
screen shot 2017-04-03 at 11 16 20 am
screen shot 2017-04-03 at 11 16 10 am

File: pegasus/sites.v3/code.org/public/college-board.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/college-board.md:1]: Same as 3rdparty-partners. The bulleted list on this page is especially small and weirdly centered as is.

File: pegasus/sites.v3/code.org/public/custom-certificates.haml [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
-Title: Print Certificates
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/custom-certificates.haml:2]: Similar to certificates, text and name entry box are off the screen to the right.

File: pegasus/sites.v3/code.org/public/diversity.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/diversity.md:1]: Green bar graph near the bottom needs to scale. 
screen shot 2017-04-03 at 11 23 30 am

File: pegasus/sites.v3/code.org/public/employee-engagement/classroom.md [hunk 6]
```diff
@@ -2,6 +2,7 @@
 title: Volunteer in a Classroom
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/employee-engagement/classroom.md:2]: Do we want the video thumbnail and the description to line up? 
screen shot 2017-04-03 at 11 25 22 am

File: pegasus/sites.v3/code.org/public/employee-engagement/index.md [hunk 7]
```diff
@@ -1,6 +1,7 @@
 ---
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/employee-engagement/index.md:1]: This page is a good candidate for thinking about responsifying tables; it's mostly only a table and the content is lost on small screens.

File: pegasus/sites.v3/code.org/public/equityDocs/index.md [hunk 8]
```diff
@@ -1,5 +1,6 @@
 ---
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/equityDocs/index.md:1]: This page has the same lime green bar graph that doesn't shrink.

File: pegasus/sites.v3/code.org/public/film/no-youtube.haml [hunk 9]
```diff
@@ -1,5 +1,6 @@
 ---
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/film/no-youtube.haml:1]: content is flush against the footer. I thought that there was space there for all the pages built in to brstrap? 
screen shot 2017-04-03 at 11 28 56 am

File: pegasus/sites.v3/code.org/public/girls.md [hunk 10]
```diff
@@ -2,6 +2,7 @@
 title: Recruit Girls to Computer Science
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/girls.md:2]: give a little bottom padding to these videos when they stack. 
screen shot 2017-04-03 at 11 30 34 am

File: pegasus/sites.v3/code.org/public/help/other.md [hunk 11]
```diff
@@ -1,5 +1,6 @@
 ---
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/help/other.md:1]: The swag marketing box doesn't fit on small screens. 
screen shot 2017-04-03 at 11 32 13 am

File: pegasus/sites.v3/code.org/public/markdown.md [hunk 12]
```diff
@@ -1,6 +1,7 @@
 ---
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/markdown.md:1]: the charts and image example on this page don't scale to small screens

File: pegasus/sites.v3/code.org/public/professional-development-workshops/index.md [hunk 13]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Find a workshop
+theme: responsive
 ---
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/professional-development-workshops/index.md:4]: the map on this page throws a JS error...
screen shot 2017-04-03 at 11 44 00 am

File: pegasus/sites.v3/code.org/public/quotes.haml [hunk 14]
```diff
@@ -1,5 +1,6 @@
 ---
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/quotes.haml:1]: more buttons squished against footer on this page 
screen shot 2017-04-03 at 11 47 07 am

File: pegasus/sites.v3/code.org/public/totm.md [hunk 15]
```diff
@@ -2,11 +2,14 @@
 title: Nominate Teacher of the Month
 nav: educate_nav
 ---
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/totm.md:4]: needs theme: responsive?

File: pegasus/sites.v3/code.org/public/visit.md [hunk 16]
```diff
@@ -1,9 +1,9 @@
 ---
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/visit.md:1]: Should the submit on this page look like a button?

**Pull request merged**